src/gapminder_challenge/dashboard/dash_app2.py

```diff
@@ -0,0 +1,80 @@
+import pandas as pd
+from dash import Dash, html, dcc, Input, Output
+import altair as alt
+
+df = pd.read_csv('../../data/raw/world-data-gapminder_raw.csv')
+
+
+url = '/dash_app2/'
+
+def add_dash(server):
+    """
+    It creates a Dash app that plots a line chart of children per woman from gapminder dataset
+    with 2 widgets : rangeslider for years and dropdown for filter
+    
+    :param server: The Flask app object
+    :return: A Dash server
+    """
+    app = Dash(server=server, url_base_pathname=url)
+
+    app.layout = html.Div([
+        html.Iframe(
+            id='line_children',
+            style={'border-width': '0', 'width': '100%', 'height': '400px'}),
+            html.Label([
+                'Zoom in years: ',
+                dcc.RangeSlider(1910, 2020, 10, value=[1910, 2020], id='year_range_slider',
+                    marks={str(year): str(year) for year in range(1910, 2030,10)}),
+            ]),
+            html.Label([
+                'See breakdown number by: ',
+                    dcc.Dropdown(options=[
+                            {'label': 'All', 'value': 'all'},
+                            {'label': 'Income Group', 'value': 'income_group'},
+                            {'label': 'Region', 'value': 'region'}
+                        ],
+                        value='', id='filter_dropdown')
```

Comment:
```suggestion
                        value='all', id='filter_dropdown')
```

I think you can set it by default so you don't need to check for empty

Reply:
It will become empty if I deselect (press the cross mark) in the dropdown.